src/test/visit_test_main.py

```diff
@@ -1594,9 +1604,11 @@ def ProcessDiffImage(case_name, baseimg, testimg, diffimg):
 #   Cyrus Harrison, Tue Nov  6 13:08:56 PST 2012
 #   Make sure to filter TestEnv.params["run_dir"] as well.
 #
+#   Mark C. Miller, Fri Sep 11 19:55:17 PDT 2020
+#   Added _numdifftol arg
 # ----------------------------------------------------------------------------
 
-def FilterTestText(inText, baseText):
+def FilterTestText(inText, baseText, _numdifftol):
```

Comment:
Is there any particular reason to have _numdifftol and numdifftol instead of just numdifftol? It looks like numdifftol is the only one ever actually used.